extension/test/e2e/fixtures/affirm-make-payment-form.html

```diff
@@ -0,0 +1,294 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Make Payment</title>
+    <link
+      rel="stylesheet"
+      href="https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/4.3.0/adyen.css"
+      integrity="sha384-5CDvDZiVPuf+3ZID0lh0aaUHAeky3/ACF1YAKzPbn3GEmzWgO53gP6stiYHWIdpB"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <div>
+      <input
+        type="text"
+        id="adyen-client-key"
+        placeholder="Adyen client key"
+      /><br />
+      <textarea id="adyen-make-payment-request">
+Make payment request will come here</textarea
+      >
+      <textarea id="adyen-payment-form-input"> </textarea>
+      <br />
+      <input type="button" id="gen-container-details" />
+      <input type="text" class="firstname secondname" />
+    </div>
+    <br />
+    <div id="affirm-container"></div>
+
+    <script
+      src="https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/4.3.0/adyen.js"
+      integrity="sha384-cNkiBPQRGouJfbstYuSccx2XkGe3RB28iYrjge6rLIDtex7fk5+3/E9f4EZ34fxE"
+      crossorigin="anonymous"
+    ></script>
+    <script type="application/javascript">
+      const configuration = {
+        environment: 'test', // When you"re ready to accept live payments, change the value to one of our live environments https://docs.adyen.com/checkout/components-web#testing-your-integration.
+        clientKey: '', // Your client key. To find out how to generate one, see https://docs.adyen.com/development-resources/client-side-authentication.
+        paymentMethodsResponse: {
+          paymentMethods: [
```

Comment:
There is no affirm in the paymentMethods.

Reply:
I obtain the payment methods again by CURL command to Adyen and update them in the make-payment form.
https://docs.adyen.com/online-payments/drop-in-web#methods-request

https://github.com/commercetools/commercetools-adyen-integration/pull/769/commits/1e82fdb3fea3b81cd2ddb89f497bab5969a66f97